spec/lib/search_spec.rb

```diff
@@ -1991,7 +1991,7 @@ def search_with_bookmarks
       expect(Search.execute("with:images").posts.map(&:id)).to contain_exactly(post_uploaded.id)
     end
 
-    it "can find by latest" do
+    it "can find by latest and oldest" do
```

Comment:
```suggestion
    it "can find by latest" do
```

I don't think we changed anything in this test that asserts we can search by olderst.

Reply:
oh yeah this was my first go but I realized merging did not make sense!